src/styles/blocks/header.scss

```diff
@@ -0,0 +1,61 @@
+@import "../utils/variables.scss";
+
+.header {
+  width: 100%;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+}
+
+.nav {
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  margin-right: 50px;
+  &__item {
+    display: inline-block;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14px;
+    padding-left: 20px;
+  }
+  &__link {
+    text-decoration: none;
+    color: $dark-grey;
+    padding: 23px 0;
+  }
+  &__link:hover {
+    color: $blue;
+  }
```

Comment:
Check the 9th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)